js/SearchResult.js

```diff
@@ -0,0 +1,54 @@
+'use strict';
+class SearchResult {
+  constructor(selector) {
+    this.elContainer = document.querySelector(selector);
+    this.resultList = document.createElement('ul');
+    this.resultList.classList.add('result-list');
+    this.elContainer.append(this.resultList);
+  }
+
+  renderSearchResult(dataArray) {
+    this.resultList.innerHTML = '';
+    const fragment = document.createDocumentFragment();
+
+    for (let i = 0; i < dataArray.length; i++) {
+      const companyLogo = document.createElement('img');
+      companyLogo.src = dataArray[i].image;
+
+      const companyPercentage = document.createElement('span');
+      companyPercentage.innerHTML = `<b>(${dataArray[i].changesPercentage})</b>`;
+      companyPercentage.style.color =
+        dataArray[i].changesPercentage < 0 ? 'red' : 'green';
+
+      let inputElement = document.getElementById('search-input');
+
+      let listElement = `${dataArray[i].name.replace(
```

Comment:
i would extract the code of milestone 7 into another function in order to make the code simpler.